docs/semgrep-app/getting-started-with-semgrep-app.md

```diff
@@ -0,0 +1,284 @@
+---
+slug: getting-started-with-semgrep-app
+append_help_link: true
+title: Getting started with Semgrep App
+description: "Get started with Semgrep App to scan for security vulnerabilities on cloud repositories hosted on GitHub and GitLab."
+---
+
+# Scanning cloud repositories with Semgrep App
+
+Semgrep App enables you to run scans on multiple repositories by integrating with your GitHub or GitLab-hosted account. Scans are powered by **rules**. A Semgrep rule encapsulates pattern-matching logic and data-flow analysis used to find code violations, security issues, outdated libraries, and other issues.

```

Comment:
"powered" in this instance strikes me as interesting. I do think rules are the *fuel* for the Semgrep engine, but not necessary what "powers" a scan.

Reply:
Adam actually gave the same note! I will fix this.

Reply:
Resolved by removing "powered" and simplifying the wording in the interests of accessibility and clarity. In hindsight, using metaphors should be avoided. Thanks.